### PR TITLE
Enable dynamo backup

### PR DESCRIPTION
**Pull request by yito88**
https://scalar-labs.atlassian.net/browse/DLT-8207

Enabled the continuous backup (for Point Time In Recovery)

Note:
This doesn't enable the backup of the `scalardb.metadata` table.
The restored metadata would be useless because the user table name should be changed.
(DynamoDB cannot restore data to the existing table.)
So, we have to remake the metadata table with this tool with the new prefix after restoring other tables.

### CODE REVIEW

**Review by brfrn169 — APPROVED** [review]
LGTM!

File: tools/scalar-schema/src/scalar_schema/core.clj [hunk 1]
```diff
@@ -24,6 +24,7 @@
    [nil "--region REGION" "Region where the tool creates tables for DynamoDB"]
    [nil "--prefix NAMESPACE_PREFIX" "Namespace prefix. The prefix is added to all the namespaces."]
    [nil "--no-scaling" "Disable auto-scaling"]
+   [nil "--no-backup" "Disable continuous backup"]
```
Comment (feeblefakie) [inline tools/scalar-schema/src/scalar_schema/core.clj:27]: Just a question. This option can only be specified for DynamoDB at the moment and would be ignored if specified for other databases?

**Review by feeblefakie — APPROVED** [review]
LGTM! 
Left a clarifying question.

File: tools/scalar-schema/src/scalar_schema/core.clj (hunk 1 above)
Comment (yito88) [reply tools/scalar-schema/src/scalar_schema/core.clj:27]: @feeblefakie Thank you for the question. Right, it will be ignored for other databases.

**Pull request merged**